### PR TITLE
Add step Then size of decompressed response body is comparisonRule sizeInBytes

### PR DESCRIPTION
**Pull request by AlenaBudevich**
… bytes

### CODE REVIEW

File: vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java [hunk 1]
```diff
@@ -139,6 +139,21 @@ public void doesResponseBodyMatchResource(ByteArrayValidationRule validationRule
...
     }
 
+    /**
+     * Compare size of uncompressed image from HTTP response with value <b>bytes</b>
+     * @param comparisonRule The rule to compare values<br>
+     * (<i>Possible values:<b> less than, less than or equal to, greater than, greater than or equal to,
+     * equal to</b></i>)
+     * @param bytes value f
+     */
+    @Then("size of uncompressed image from response $comparisonRule `$bytes`")
```
Comment (ikalinin1) [inline vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java]: bytes looks confusing, it is the length

Comment (uarlouski) [inline vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java]: image > smth like body?

Comment (valfirst) [inline vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java]:
```
@Then("size of uncompressed response body is $comparisonRule `$sizeInBytes`")
```

File: vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java [hunk 2]
```diff
@@ -139,6 +139,21 @@ public void doesResponseBodyMatchResource(ByteArrayValidationRule validationRule
                 ResourceUtils.loadResourceAsByteArray(getClass(), resourcePath), response.getResponseBody()));
     }
 
+    /**
+     * Compare size of uncompressed image from HTTP response with value <b>bytes</b>
+     * @param comparisonRule The rule to compare values<br>
+     * (<i>Possible values:<b> less than, less than or equal to, greater than, greater than or equal to,
+     * equal to</b></i>)
+     * @param bytes value f
```
Comment (valfirst) [inline vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java]: should be updated

File: vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java [hunk 3]
```diff
@@ -139,6 +139,21 @@ public void doesResponseBodyMatchResource(ByteArrayValidationRule validationRule
                 ResourceUtils.loadResourceAsByteArray(getClass(), resourcePath), response.getResponseBody()));
     }
 
+    /**
+     * Compare size of uncompressed HTTP response body with value <b>sizeInBytes</b>
+     * @param comparisonRule The rule to compare values<br>
+     * (<i>Possible values:<b> less than, less than or equal to, greater than, greater than or equal to,
+     * equal to</b></i>)
+     * @param sizeInBytes value for comparison
```
Comment (valfirst) [inline vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java]: `value for comparison` -> `expected size of the response body in bytes`

File: vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java [hunk 4]
```diff
@@ -139,6 +139,21 @@ public void doesResponseBodyMatchResource(ByteArrayValidationRule validationRule
...
+     */
+    @Then("size of uncompressed response body is $comparisonRule `$sizeInBytes`")
+    public void doesUncomressedResponseBodySizeConfirmRule(ComparisonRule comparisonRule, int sizeInBytes)
+    {
+        performIfHttpResponseIsPresent(response ->
+            softAssert.assertThat("Size of uncompressed image from HTTP response",
```
Comment (valfirst) [inline vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpResponseValidationSteps.java]: `Size of uncompressed HTTP response body`

**Pull request merged**